lib/messaging/messagehandler.rb

```diff
@@ -0,0 +1,151 @@
+require 'json'
+require 'time'
+require './lib/database/dbaccess.rb'
+
+require_relative 'messagehandler_emergencystop.rb'
+require_relative 'messagehandler_logs.rb'
+require_relative 'messagehandler_measurements.rb'
+require_relative 'messagehandler_message.rb'
+require_relative 'messagehandler_parameters.rb'
+require_relative 'messagehandler_schedule.rb'
+require_relative 'messagehandler_status.rb'
+
+
+# Get the JSON command, received through skynet, and send it to the farmbot
+# command queue Parses JSON messages received through SkyNet.
+class MessageHandler
+
+  attr_accessor :message
+  attr_accessor :use_test_handler
+
+  ## general handling messages
+
+  def initialize
+    #@dbaccess = DbAccess.new
+    @dbaccess = $dbaccess
```

Comment:
Was this a copy/paste job from an older commit? Looks like the $dbaccess global variable is back after being removed.